package.json

```diff
@@ -8,8 +8,38 @@
       "es2015"
     ]
   },
-  "dependencies": {},
-  "devDependencies": {},
+  "dependencies": {
+    "async": "^2.0.1",
+    "blipp": "^2.3.0",
+    "camelcase": "^3.0.0",
+    "confidence": "^3.0.1",
+    "glob": "^7.1.0",
+    "glue": "^4.0.0",
+    "good": "^7.0.2",
+    "good-console": "^6.1.2",
+    "good-file": "^6.0.1",
+    "good-squeeze": "^5.0.0",
+    "hapi": "^15.0.3",
+    "hapi-swagger": "^7.2.0",
+    "inert": "^4.0.2",
+    "lodash": "^4.16.1",
+    "scooter": "^4.0.0",
+    "vision": "^4.1.0"
+  },
+  "devDependencies": {
+    "babel-cli": "^6.14.0",
+    "babel-core": "^6.14.0",
+    "babel-eslint": "^6.1.2",
+    "babel-preset-es2015": "^6.14.0",
+    "code": "^4.0.0",
+    "eslint": "^3.5.0",
+    "eslint-config-hapi": "^10.0.0",
+    "eslint-plugin-hapi": "^4.0.0",
+    "fixpack": "^2.3.1",
+    "label": "^0.2.2",
```

Comment:
@aramonc `label` isn't the test runner the package is `lab` sorry maybe was my mistake.